### ISSUE TITLE
soql:query shows [object] for json fields

### ISSUE THREAD

**Issue by mshanemc** [forcedotcom/cli#1318]
### Summary

Another way to get [object Object] queries

### Steps To Reproduce:

1. in an org, create a RemoteSiteSetting in the UI.
2. sfdx force:data:soql:query -t -q "select id, Metadata from RemoteProxy"

### Expected result

See the Metadata fields as a string that is json
 
### Actual result
```
ID                  METADATA
──────────────────  ───────────────
0rp2F000000CH6tQAG  [object Object]
0rp2F000000EJ1IQAW  [object Object]
Total number of records retrieved: 2.
Querying Data... done
```
### System Information

```
sfdx version --verbose
 CLI Version : 
        sfdx-cli/7.130.1

 Architecture: 
        darwin-x64

 Node Version : 
        node-v16.13.1

 Plugin Version: 
        @mshanemc/sfdx-sosl 1.1.0
        @oclif/plugin-autocomplete 0.3.0 (core)
        @oclif/plugin-commands 1.3.0 (core)
        @oclif/plugin-help 3.2.10 (core)
        @oclif/plugin-not-found 1.2.4 (core)
        @oclif/plugin-plugins 1.10.6 (core)
        @oclif/plugin-update 1.5.0 (core)
        @oclif/plugin-warn-if-update-available 1.7.3 (core)
        @oclif/plugin-which 1.0.3 (core)
        @salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
        alias 1.2.0 (core)
        apex 0.4.0 (core)
        auth 1.8.0 (core)
        config 1.3.0 (core)
        custom-metadata 1.0.12 (core)
        data 0.6.6 (core)
        generator 1.2.1 (core)
        info 1.1.2 (core)
        limits 1.3.0 (core)
        org 1.11.0 (core)
        release-management 2.7.2
        └─ trust 1.1.0
        salesforce-alm 53.6.0 (core)
        schema 1.1.0 (core)
        sfdx-cli 7.130.1 (core)
        sfdx-migration-automatic 4.2.3
        shane-sfdx-plugins 4.43.0
        └─ @mshanemc/plugin-streaming 1.1.7
        source 1.6.2 (core)
        telemetry 1.4.0 (core)
        templates 52.6.0 (core)
        user 1.7.0 (core)

 OS and Version: 
        Darwin 19.6.0
```

## Additional Information

here's the actual json response for that field
```
"Metadata": {
          "disableProtocolSecurity": false,
          "isActive": true,
          "url": "https://google.com",
          "urls": null,
          "description": "test"
        }
```

**Comment by uip-robot-zz** [comment]
This issue has been linked to a new work item: [W-10290520](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000Wj3CQYAZ/view)

**Comment by iowillhoit** [comment]
Fixed and released in `sfdx 7.138.1`
